### ISSUE TITLE
iOS13 b5 Crashes when setting an attributed placeholder

### ISSUE THREAD

**Issue by iamcam** [devxoul/UITextView-Placeholder#63]
When setting an attributed placeholder in iOS13 b5, a crash occurs at `dispatch_once` in UITextView+Placeholder.m

```
+ (UIColor *)defaultPlaceholderColor {
    static UIColor *color = nil;
    static dispatch_once_t onceToken;
    dispatch_once(&onceToken, ^{
        UITextField *textField = [[UITextField alloc] init];
        textField.placeholder = @" ";
        color = [textField valueForKeyPath:@"_placeholderLabel.textColor"];
    });
    return color;
}
```

For example, in our Swift code,
```
let attributes: [NSAttributedString.Key: Any] = [NSAttributedString.Key.font: font,
                                                            NSAttributedString.Key.kern: kern,
                                                            NSAttributedString.Key.foregroundColor: Style.placeholderColor]

 textView.attributedPlaceholder = NSAttributedString(string: "Description", attributes: attributes)
```

**Comment by fl034** [comment]
Duplicate of https://github.com/devxoul/UITextView-Placeholder/issues/61

**Comment by devxoul** [comment]
Fixed in #64
